src/VirtualClient/VirtualClient.Main/profiles/PERF-CPU-OPENSSL.json

```diff
@@ -11,7 +11,7 @@
             "Type": "OpenSslExecutor",
             "Parameters": {
                 "Scenario": "MD5",
-                "CommandArguments": "speed -elapsed -seconds 100 md5",
+                "CommandArguments": "speed -elapsed -seconds 100 md54321",
```

Comment:
Is this intentional?

Reply:
No. Forgot to change it back but just did and pushed the change again. Good eyes!!